lib/Sema/TypeCheckAttr.cpp

```diff
@@ -272,7 +272,7 @@ void AttributeChecker::visitTransparentAttr(TransparentAttr *attr) {
     if (!(isa<AccessorDecl>(D) && D->isImplicit()))
       diagnoseAndRemoveAttr(attr, diag::transparent_in_classes_not_supported);
   }
-  
+
```

Comment:
Note: there's some whitespace changes here in non-autodiff-code. Seems fine though.

Reply:
Thanks for noticing! The whitespace changes existed on `tensorflow` branch, so I upstreamed them as is. Sorry for not clarifying.